docs/design/canvas/OverviewScale.dc.html

```
@@ -0,0 +1,102 @@
+      All repos <span style="color:#5A6377;">&#9662;</span></div>
+    <div style="display:flex; gap:4px;"><span style="padding:4px 10px; font-size:11px; border:1px solid #7A9BEF88; background:#7A9BEF1a; color:#7A9BEF; font-weight:600; border-radius:4px;">Last 50 rounds / 7d</span><span style="padding:4px 10px; font-size:11px; border:1px solid #1E2433; color:#8B93A7; border-radius:4px;">30d</span><span style="padding:4px 10px; font-size:11px; border:1px solid #1E2433; color:#8B93A7; border-radius:4px;">90d</span><span style="padding:4px 10px; font-size:11px; border:1px solid #1E2433; color:#8B93A7; border-radius:4px;">All time</span></div>
+  </div>
+    <div style="flex:1; padding:18px 20px; display:flex; flex-direction:column; gap:14px; min-width:0;">
+    <div style="display:flex; gap:12px; align-items:stretch;">
+    <div style="flex:1.15; display:flex; flex-direction:column; gap:12px; min-width:0;">
+      <div style="display:flex; gap:12px;">
+        <div style="flex:1; background:#121623; border:1px solid #1E2433; border-radius:6px; padding:14px 16px; display:flex; flex-direction:column; gap:5px; min-width:0;">
+    <div style="font-size:11px; font-weight:600; color:#8B93A7; letter-spacing:0.04em; text-transform:uppercase;">PRs reviewed</div>
+    <div style="display:flex; align-items:baseline; gap:8px;"><span class="num" style="font-size:34px; font-weight:500; letter-spacing:-0.02em;">1,834</span></div>
+    <div style="font-size:11px; color:#8B93A7;">this week · 262/day mean</div>
+    <div style="font-size:10.5px; color:#5A6377;">the per-day rate line renders at >= 30 rounds in window; below that the all-time cumulative renders instead</div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)